### DISCUSSION TITLE
Need Support to verify my teacher statues

### DISCUSSION THREAD

**Discussion by XuyangLu** [community/community#54224] [GitHub Education]
### Hi everyone,

I have requested a formal letter from the university and uploaded it. I need some additional guidance here.

**Comment by samus-aran** [comment]
Hi @XuyangLu  👋

We'd love to help! You are more likely to get a useful response if you are explicit about the feedback given to you from the rejected application submission. Do you have a screenshot of the rejection reasons? This might help someone give you a nudge in the right direction.

**Reply by XuyangLu** [reply]
![image](https://user-images.githubusercontent.com/86523565/236133183-155dbf88-0f63-496b-82ca-c9591f42f881.png)

Here are the rejected reasons. I have submitted a letter from our university to prove my faculty status. 
The following is my google scholar page. 
https://scholar.google.com/citations?hl=en&user=dcD7BK0AAAAJ&view_op=list_works&sortby=pubdate
